README.md

```diff
@@ -1,2 +1,139 @@
+3. A workflow file which runs the two actions on a schedule, continually creating new tickets when necessary.
+
+
+### Jira API Token
+The `reload/github-security-jira` action requires a secret called `JiraApiToken` which should contain an API Token for the Jira user that should be used to create tickets. The token is passed to the action in the `JIRA_TOKEN` environment variable.
+
+
+### Jira config file `gh-jira.yaml`
+The repo should also contain a config file for Jira called `gh-jira.yaml`. It has some required and optional settings. It allows you to configure the Jira hostname, user and project, as well as ticket type, who to set as watchers on the ticket and an additional, restricted comment to post on the ticket (for eg. developers only).
+
+Here is an example `gh-jira.yaml` file:
+```yaml
+# Jira ticket settings for this repo.
+
+# REQUIRED: The endpoint for your Jira instance's issue API, e.g.
+# https://foo.atlassian.net
+jira_host: 'https://foo.atlassian.net'
+
+# REQUIRED: The ID of the Jira user which is associated with the 'JiraApiToken' env
+# variable.
+jira_user: 'example@reload.dk'
+
+# REQUIRED: The identifier key for the Jira project you want to create issues for.
+jira_project_key: 'TEST'
+
+# OPTIONAL: Jira users to add as watchers to tickets.
+# jira_watchers:
+#   - 'email@example.com'
+#   - 'otheremail@example.com'
+
+# OPTIONAL: Type of ticket to create.
+# jira_issue_type: 'Security'
+
+# OPTIONAL: Jira group for restricted comments.
+# jira_restricted_group: 'Developers'
+
+# OPTIONAL: Text for restricted comment. Use the chomping indicator (pipe symbol) for
+# multiline texts.
+# jira_restricted_comment: |
+#   Remember to evaluate severity here and set ticket priority.
+#   Check out the guide [in our wiki|https://foo.atlassian.net/wiki/]!
+```
+
+
+### Workflow file setup
+The [GitHub workflow file](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/configuring-a-workflow#creating-a-workflow-file) should reside in any repo where you want to sync security alerts with Jira.
+
+The workflow file will need a secret called `GitHubSecurityToken` in the repository. This secret should contain a [Personal Access Token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) for the GitHub user under which this action should be executed, and the token must include the `public_repo` scope. The token is passed to the action in the `GITHUB_TOKEN` environment variable. Also, the GH user must have access to the repository and [access to security alerts in the repo](https://help.github.com/en/github/managing-security-vulnerabilities/managing-alerts-for-vulnerable-dependencies-in-your-organization).
+
+The workflow uses another GitHub action `octokit/graphql-action` and its output when used to query [GitHub's GraphQL API](https://developer.github.com/v4/).
```

Comment:
Isn't there a Symfony GraphQL package? Having to configure a different action just to get the data (with the danger of messing that up) seems a bit overwhelming.

Reply:
I agree. With the change above, a minimum workflow would become:
```yaml
jobs:
  syncSecurityAlerts:
    name: "Get security alerts for this repo"
    runs-on: ubuntu-latest
    steps:
      - name: "Sync alert data to Jira tickets"
        uses: reload/github-security-jira@v1.x
        env:
          GITHUB_TOKEN: ${{ secrets.GitHubSecurityToken }}
          JIRA_TOKEN: ${{ secrets.JiraApiToken }}
          JIRA_HOST: <hostname>
          JIRA_USER: <username>
          JIRA_PROJECT: <project id>
```

in any repo that wants to use this action.

Agreed?

Reply:
I concur!

Reply:
@achton 
Totally